exercises/connect/connect_test.rb

```diff
@@ -6,93 +6,73 @@
 class ConnectTest < Minitest::Test
   EMPTY = [
     '. . . . .',
-    ' . . . . .',
-    '  . . . . .',
-    '   . . . . .',
-    '    . . . . .'
+    '. . . . .',
+    '. . . . .',
+    '. . . . .',
+    ' . . . . .'
   ].freeze
 
   EDGES_POPULATED_BUT_NO_WINNER = [
     '. . . . .',
-    ' . . . . X',
-    '  . O . X .',
-    '   . . O . .',
-    '    . . . O .'
+    '. . . . X',
+    '. O . X .',
+    '. . O . .',
+    '. . . O .'
   ].freeze
 
   RECTANGLE_BLACK = [
     '. O . .',
-    ' O X X X',
-    '  O X O .',
-    '   X X O X',
-    '    . O X .'
+    'O X X X',
+    'O X O .',
+    'X X O X',
+    '. O X .'
   ].freeze
 
   RECTANGLE_WHITE = [
     '. O . .',
-    ' O X X X',
-    '  O O O .',
-    '   X X O X',
-    '    . O X .'
+    'O X X X',
+    'O O O .',
+    'X X O X',
+    '. O X .'
   ].freeze
 
   CONVOLUTED_PATH = [
     '. X X . .',
-    ' X . X . X',
-    '  . X . X .',
-    '   . X X . .',
-    '    O O O O O'
+    'X . X . X',
+    '. X . X .',
+    '. X X . .',
+    'O O O O O'
   ].freeze
 
   ASPIRAL = [
-    ' O X X X X X X X X',
-    '  O X O O O O O O O',
-    '   O X O X X X X X O',
-    '    O X O X O O O X O',
-    '     O X O X X X O X O',
-    '      O X O O O X O X O',
-    '       O X X X X X O X O',
-    '        O O O O O O O X O',
-    '         X X X X X X X X O'
+    'O X X X X X X X X',
+    'O X O O O O O O O',
+    'O X O X X X X X O',
+    'O X O X O O O X O',
+    'O X O X X X O X O',
+    'O X O O O X O X O',
+    'O X X X X X O X O',
+    'O O O O O O O X O',
+    'X X X X X X X X O'
   ].freeze
 
-  def test_empty_board_has_no_winner
-    board = Board.new EMPTY
-    assert_equal board.winner, ''
-  end
-
-  def test_populated_but_no_winner
-    board = Board.new EDGES_POPULATED_BUT_NO_WINNER
-    assert_equal board.winner, ''
-  end
-
-  def test_1x1_board_wins_black
-    board = Board.new ['X']
-    assert_equal board.winner, 'X'
-  end
-
-  def test_1x1_board_wins_white
-    board = Board.new ['O']
-    assert_equal board.winner, 'O'
-  end
-
-  def test_convoluted_path
-    board = Board.new CONVOLUTED_PATH
-    assert_equal board.winner, 'X'
-  end
-
-  def test_rectangle_white_wins
-    board = Board.new RECTANGLE_WHITE
-    assert_equal 'O', board.winner
-  end
-
-  def test_rectangle_black_wins
-    board = Board.new RECTANGLE_BLACK
-    assert_equal 'X', board.winner
-  end
+  TEST_CASES = [
+    [EMPTY, '', 'an empty board has no winner'],
+    [EDGES_POPULATED_BUT_NO_WINNER, '', 'only edges does not make a winner'],
+    [['X'], 'X', 'a one cell board with a X should get the winner easy'],
+    [['O'], 'O', 'a one cell board with a O should get the winner easy'],
+    [[' . '], '', 'a one cell board empty has no winner'],
+    [CONVOLUTED_PATH, 'X', 'Convoluted path x wins'],
+    [RECTANGLE_BLACK, 'X', 'Rectangle black - black wins'],
+    [RECTANGLE_WHITE, 'O', 'Rectangle white - white wins'],
+    [ASPIRAL, 'X', 'Aspiral black wins']
+  ].freeze
```

Comment:
I like that they are together like this, but if you have done the exercism submissions, you will notice that the tests are broken out to individual steps, and each one skipped, so that the failures aren't overwhelming.

They should be as they were before.  They also should have a skip call after each test definition.